builder/role_image.go

```diff
@@ -372,12 +355,27 @@ func (j roleBuildJob) Run() {
 			return fmt.Errorf("Error calculating checksum for role %s: %s", j.role.Name, err.Error())
 		}
 		roleImageName := GetRoleDevImageName(j.repository, j.role, devVersion)
+		outputPath := filepath.Join(j.outputDirectory, fmt.Sprintf("%s.tar", roleImageName))
 		if !j.force {
-			if hasImage, err := j.dockerManager.HasImage(roleImageName); err != nil {
-				return err
-			} else if hasImage {
-				j.ui.Printf("Skipping build of role image %s because it exists\n", color.YellowString(j.role.Name))
-				return nil
+			if j.outputDirectory == "" {
+				if hasImage, err := j.dockerManager.HasImage(roleImageName); err != nil {
+					return err
+				} else if hasImage {
+					j.ui.Printf("Skipping build of role image %s because it exists\n", color.YellowString(j.role.Name))
+					return nil
+				}
+			} else {
+				info, err := os.Stat(outputPath)
+				if err == nil {
+					if info.IsDir() {
```

Comment:
Is this enough? What if it's an empty directory? Failed compile or something. Just asking the question.